src/modules/viewmodel_sway.rs

```diff
@@ -0,0 +1,127 @@
+//! Viewmodel sway
+use std::f32::consts::PI;
+
+use super::Module;
+use crate::hooks::engine::{self, ref_params_s};
+use crate::modules::cvars::CVar;
+use crate::utils::*;
+
+pub struct ViewmodelSway;
+impl Module for ViewmodelSway {
+    fn name(&self) -> &'static str {
+        "Viewmodel sway"
+    }
+
+    fn description(&self) -> &'static str {
+        "Viewmodel sway."
+    }
+
+    fn cvars(&self) -> &'static [&'static CVar] {
+        static CVARS: &[&CVar] = &[
+            &BXT_VIEWMODEL_SWAY,
+            &BXT_VIEWMODEL_SWAY_MAX,
+            &BXT_VIEWMODEL_SWAY_RATE,
+        ];
+        CVARS
+    }
+
+    fn is_enabled(&self, marker: MainThreadMarker) -> bool {
+        engine::ClientDLL_CalcRefdef.is_set(marker) && engine::cl_viewent.is_set(marker)
+    }
+}
+
+static BXT_VIEWMODEL_SWAY: CVar = CVar::new(
+    b"bxt_viewmodel_sway\0",
+    b"0\0",
+    "\
+Setting to `1` enables weapon sway. Default is `100` max and `0.02` rate.",
+);
+
+static BXT_VIEWMODEL_SWAY_MAX: CVar = CVar::new(
+    b"bxt_viewmodel_sway_max\0",
+    b"100\0",
+    "\
+The max of sway.",
```

Comment:
This is not how english works im pretty sure

Reply:
I am too never fond of article.